packages/the-org/src/projectManager/plugins/team-coordinator/actions/addTeamMemberAction.ts

```
@@ -0,0 +1,629 @@
+        // Fetch the updated team members to include in the response
+        const updatedMemories = await runtime.getMemories({
+          roomId: roomIdForStoringTeamMembers as UUID,
+          tableName: 'messages',
+        });
+
+        const updatedConfig = updatedMemories.find(
+          (memory) => memory.content.type === 'store-team-members-memory'
+        );
+
+        if (updatedConfig && updatedConfig.content.config?.teamMembers) {
+          const allTeamMembers = updatedConfig.content.config.teamMembers;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)